tests/test_uniswap.py

```diff
@@ -474,13 +476,6 @@ def test_make_trade(
             # ("ETH", "UNI", int(0.000001 * ONE_ETH), ZERO_ADDRESS),
             # ("UNI", "ETH", int(0.000001 * ONE_ETH), ZERO_ADDRESS),
             # ("DAI", "UNI", int(0.000001 * ONE_ETH), ZERO_ADDRESS),
-            (
```

Comment:
I fixed the failing tests, though I ask you to take a look at the deleted test line. It was failing because `InsufficientBalance` wasn't raised during the tests.

Reply:
Strange, I will add this back in a new PR and sort it out later.